packages/aragon-cli/src/lib/configEnvironment.ts

```diff
@@ -0,0 +1,203 @@
+      host?: string // 'localhost',
+      port?: number // 8545,
+      provider?: HDWalletProviderType | (() => HDWalletProviderType)
+      gas: number // 6.9e6,
+      gasPrice: number // 15000000001;
+    }
+  }
+}
+
+interface ApmConfig {
+  ipfs: { gateway: string }
+}
+
+interface Argv {
+  // Options from commands
+  _: string[]
+  gasPrice: string
+  silent: boolean
+  debug: boolean
+  cwd: string
+  useFrame: boolean
+  network?: string // The network in your truffle.js that you want to use
+  environment?: string // The environment in your arapp.json that you want to use
+  apm: ApmConfig
+  // Objects injected from other middlewares
+  module?: ArappModule
+  reporter: any
+}
+
+interface EnvironmentsConfig {
+  [environmentName: string]: EnvironmentConfig
+}
+
+interface EnvironmentConfig {
+  registry: string // Registry contract address
+  appName?: string // ENS domain of the app
+  network: string // Network to use
+  wsRPC: string // Node WS URL
+  apm?: ApmConfig // "gateway": "https://ipfs.eth.aragon.network/ipfs"
+}
+
+interface AragonAppConfigurationFile {
+  path?: string // The path to the main contract in your app.
+  roles?: {
+    // An array of all the roles that your app has.
+    id: string // The identifier of the role as it is defined in the contract.
+    name: string // A description of the role in the app.
+    params: string[] // The names of any parameters for the role.
+  }[]
+  environments: EnvironmentsConfig // An object containing deploy environment configurations.
+}
+
+// Mutations
+interface ArappModule extends AragonAppConfigurationFile {
+  env?: EnvironmentConfig // Choosen configuration
+  appName?: string
+}
+
+function getEnv(arapp?: ArappModule, environment?: string): EnvironmentConfig {
+  if (arapp) {
+    if (!environment) environment = 'default'
+    const env = arapp.environments[environment]
+    if (!env)
+      throw Error(
+        `${environment} environment was not defined in your arapp.json.`
+      )
+    return env
+  }
+
+  // if there is no arapp.json and the command is not init default to the "global" config
+  // designed for the dao commands including dao acl
+  else {
+    if (!environment) environment = 'aragon:local'
+    const env = defaultEnvironments[environment]
+    if (!env)
+      throw Error(
+        `Could not find the ${environment} environment. Try using aragon:local, aragon:rinkeby or aragon:mainnet.`
+      )
+    return env
+  }
+}
+
+function configureNetwork(
+  network: string,
+  truffleConfig: TruffleConfig,
+  options?: { useFrame?: boolean; ignoreNetwork: boolean }
+) {
+  const { useFrame, ignoreNetwork } = options || {}
+
+  if (ignoreNetwork) return {}
+
+  if (useFrame) {
+    return {
+      name: `frame-${network}`,
+      provider: new Web3.providers.WebsocketProvider(
+        FRAME_ENDPOINT,
+        { headers: { origin: FRAME_ORIGIN } } // Provider options
+      ),
+    }
+  }
+
+  const truffleNetwork = truffleConfig.networks[network]
```

Comment:
I get this:
```
/home/daniel/r/ar/aragon-cli/packages/aragon-cli/src/lib/configEnvironment.js:50
  const truffleNetwork = truffleConfig.networks[network]
                                       ^
TypeError: Cannot read property 'networks' of null
    at networks (/home/daniel/r/ar/aragon-cli/packages/aragon-cli/src/lib/configEnvironment.js:50:40)
```
when running `node dist/cli.js devchain`

Reply:
@0x6431346e Thanks for testing it! I've committed https://github.com/aragon/aragon-cli/pull/893/commits/39303871c2b4256bb5e94e12f6bd1e4a9ecf9c52 which fixes the problem. I was able to reproduce the error locally and with this commit it works. Please if you can, confirm the issue is fixed in your local env.

> The issue was that the argument `truffleConfig` was `null` instead of `undefined` so it did not trigger the default value.